.github/workflows/static_code_checks.yaml

```diff
@@ -41,10 +41,7 @@ jobs:
         uses: pypa/gh-action-pip-audit@v1.1.0
         with:
           virtual-environment: .venv/
-          # Ignoring vulnerability that just popped up in pytorch
-          # versions < 2.2.0, GHSA-pg7h-5qx3-wjr3
-          # Poetry currently resolves torch version 2.1.2, not sure why, some
-          # other package must be preventing the pytorch version from being
-          # upgraded
+          # Ignoring vulnerability that for cryptography
```

Comment:
The two lines of this comment are slightly hard to read. Maybe clean it up slightly?

Reply:
yeah hahah this is definitely confusing, will update!